src/main/java/qna/domain/Answer.java

```diff
@@ -31,19 +29,13 @@ public class Answer {
     @Column(name = "deleted", nullable = false)
     private boolean deleted = false;
 
-    @Column(name = "writer_id")
-    private Long writerId;
+    @ManyToOne
+    @JoinColumn(name = "writer_id")
```

Comment:
```suggestion
    @JoinColumn
```

name이 없으면 어떻게 동작할지 학습해보아요.